src/runtime/HalideRuntime.h

```diff
@@ -765,4 +800,82 @@ extern double halide_float16_bits_to_double(uint16_t);
 } // End extern "C"
 #endif
 
+#ifdef __cplusplus
```

Comment:
This C++ block looks like useful syntactic sugar to me. I think it's worth having in HalideRuntime.h